src/Workspaces/Remote/ServiceHub/Host/AssetProvider.cs

```diff
@@ -248,48 +228,15 @@ Task SynchronizeProjectAssetCollectionAsync<TAsset>(AssetPath assetPath, Func<Pr
             foreach (var projectChecksums in allProjectChecksums)
                 addAllChecksums(projectChecksums, checksums, arg);
 
-            await SynchronizeAssetsAsync<TAsset>(assetPath, checksums).ConfigureAwait(false);
+            await this.GetAssetsAsync<TAsset>(assetPath, checksums, cancellationToken).ConfigureAwait(false);
         }
-
-        async Task SynchronizeProjectDocumentsAsync(ProjectStateChecksums projectChecksums)
```

Comment:
this moved to the base class.  it's used in multiple places now.  both when seeign that a project's documents changed, and also when doing hte initial bulk sync and makigna  ProjectInfo.